crowd_anki/config/config_dialog.py

```diff
@@ -0,0 +1,76 @@
+"""
+Config dialog entry point.
+This module will load the config initially.
+Write actions will be triggered here when the user clicks the dialog "Save All" button.
+"""
+
+from aqt.qt import *
+
+from .config_ui import Ui_Dialog as ConfigUI
+from .config_settings import ConfigSettings
+
+
+class ConfigDialog(QDialog):
+    def __init__(self, parent=None):
+        super().__init__(None)
+        self.parent = parent
+        self.config = ConfigSettings.get_instance()
+        self.form = ConfigUI()
+        self.form.setupUi(self)
+        self.ui_initial_setup()
+
+    def accept(self):
+        invalid_values = self.config.find_invalid_config_values()
+        if invalid_values:
+            error_message_popup = QMessageBox(self)
+            message_text = "\" , \"".join(invalid_values)
+            error_message_popup.setText(f"Invalid config values: \"{message_text}\"")
+            error_message_popup.open()
+        else:
+            self.config.save()
+            super().accept()
+
+    def ui_initial_setup(self):
```

Comment:
this one vaguely bothers me seems like a lot of boilerplate, wonder if there is a better way (I don't know one straight away, but...)